### DISCUSSION TITLE
[Announment] Welcome new committer: ZhangJian He

### DISCUSSION THREAD

**Discussion by nobodyiam** [apolloconfig/apollo#4864] [Announcements]
Hi Community,

On behalf of the Apollo PMC, I am pleased to announce that ZhangJian He a.k.a @hezhangjian , has been voted in as Apollo committer.

Please join me to say congratulations to him!

Best Regards,
Jason

**Comment by xiang092689** [comment]
👍 👍 👍

**Comment by xuthus5** [comment]
👍 👍 👍

**Comment by cyruslo** [comment]
👍 👍 👍

**Comment by fu-turer** [comment]
👍 👍 👍

**Comment by nisiyong** [comment]
Congratulations! 👏👏👏 @Shoothzj

**Comment by JaredTan95** [comment]
Welcome aboard~ @Shoothzj

**Comment by Anilople** [comment]
Welcome~ @Shoothzj
